src/server/dao/cc-proxy/SkillDao.ts

```diff
@@ -24,7 +24,17 @@ export class SkillDao extends BaseDao<Model> {
     }
 
     public async get(id: string, parameters?: { [key: string]: any }): Promise<Model> {
-        return this.getSkill(id);
+        let snapshotId: string;
+        if (!id.includes('@')) {
+            throw new ClientErrorException('The parameter `spineTreeId` is required because the given identifier is not fully qualified!');
+        }
+        snapshotId = id.substring(0, id.indexOf('@'));
+        id = id.substring(id.indexOf('@') + 1);
```

Comment:
Instead of the two calls to `substring()`, can we have:
 ```javascript
const [snapshotId, skillId] = id.split('@');
const skill: Model = await this.getSkill(snapshotId, skillId);
```
Then the given parameter `id` is left unchanged, and the code is more compact.